specificatie/openapi.yaml

```diff
@@ -557,6 +612,78 @@ components:
         pattern: '^[0-9]{16}$'
         example: '0226010000038820'
 
+    postcode:
```

Comment:
Volgens mij hebben we afgesproken dat query-parameters de post-fix "-query" in de naam krijgen om onderscheid te kunnen maken met path-parameters op dezefde property. Dus postcode-query.

Reply:
De API spec is in een eerder stadium goedgekeurd in een andere PR. Deze opmerking had ik dan bij de review van die PR verwacht. Om dit nu nog aan te passen betekent erg veel effort: API spec aanpassen, code aanpassen en tenminste 5 PR-en bijwerken en merge conflicten oplossen. Dat kost weer zeker een dag om het goed te krijgen. Die effort is het dan eigenlijk niet meer waard, zeker omdat we na volgende week donderdag (16-9-2021) hetzelfde weer moeten doen. In dat geval kunnen we w..m.b. beter alles in develop verzamelen en één 1.3 release maken zodra alle contour zoeken functionaliteit is geïmplementeerd.
Bovendien heeft het geen nut om een postfix -query achter een query parameter naam te zetten, voor iets waar toch nooit een path parameter equivalent van zal komen. Postcode is nl. geen uniek identificerend gegeven, net als huisnummer, huisletter, huisnummertoevoeging. Die uitzondering zie ik dan graag toegevoegd in de design decisions.